lang/cs/Org.Apache.REEF.Client/AzureBatch/Util/LinuxCommandBuilder.cs

```diff
@@ -1,50 +0,0 @@
-﻿// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Why are we making the assumption that there is not Linux command builder? .Net Core can run on Linux.

Reply:
will keep it and put Exception()